index.html

```diff
@@ -159,6 +171,15 @@
     }
 }
 
+toggledarkmode = function(enabled){
+  if(enabled){
+    $('body').addClass('dark-mode');

```

Comment:
Maybe we could use something like:
 
    $('body').toggleClass('dark-mode', enabled);

Untested, but [the doc](http://api.jquery.com/toggleclass/) seems to support it.

Reply:
Sure, that would be way better. I will test it.